src/PhpSpreadsheet/Writer/Xls/Xf.php

```diff
@@ -515,11 +522,6 @@ private static function mapLocked($locked)
      */
     private static function mapHidden($hidden)
     {
-        switch ($hidden) {
-            case Protection::PROTECTION_PROTECTED:
-                return 1;
-        }
-
-        return 0;
+        return array_key_exists($hidden, self::HIDDEN_ARRAY) ? HIDDEN_ARRAY[$hidden] : 0;
     }
```

Comment:
`self::HIDDEN_ARRAY`

But yes, I think the map makes it explicit for readability

Reply:
See above regarding erroneous commit dc4f568 replaced by correct dc908f5 and 9c279fb.